src/library/itunes/itunesmacosimporter.mm

```diff
@@ -89,6 +89,17 @@ int persistentIdToDbId(NSNumber* persistentId) {
         }
     }
 
+    QString uniquifyPlaylistName(QString name) {
+        auto existing = m_playlistDuplicatesByName.find(name);
+        if (existing != m_playlistDuplicatesByName.end()) {
+            m_playlistDuplicatesByName[name] += 1;
+            return QString("%1 #%2").arg(name).arg(m_playlistDuplicatesByName[name]);
+        } else {
+            m_playlistDuplicatesByName[name] = 1;
```

Comment:
I don't think we have one duplicate here.
Can we start with zero here, to have a better match with the numbers of duplicates?

Reply:
How can we have duplicates by the way? 
Maybe you can add a brief comment.

Reply:
If the user has multiple playlists with the same name in iTunes then we need to disambiguate them, because the `TreeItem` identifies the playlist by name.

> I don't think we have one duplicate here.

Yeah, the naming is not ideal yet, since we are counting the number of occurrences of a playlist name, not the duplicates, which would be one less (strictly speaking). Perhaps `m_playlistCountsByName` or similar would he better, but I wanted to make it obvious that we were going to use it for duplicate detection.

The idea is, if the user e.g. has three playlists named `Pop` that they are named

- `Pop`
- `Pop #2`
- `Pop #3`

which is similar to what the iTunes XML importer does.

Reply:
I've updated it to count from zero and added an explanatory comment.